PREPARATION.md

```diff
@@ -22,10 +22,20 @@ For general deployment, the SCIM bridge requires these components:
 
 You'll need to create a DNS record with your SCIM bridge domain name after the bridge is deployed. Follow the steps in each respective deployment guide to finish setting up your DNS record at the appropriate time.
 
+> **Note**
+>
+>Container-as-a-Service platforms (for example, Azure Container Apps, DigitalOcean App Platform) include TLS endpoint management, so you do not need to create a DNS record for these deployments.
+>
+
 ### TLS certificate
 
 Identity providers typically require a TLS certificate when communicating with the SCIM bridge. By default, TLS certificates are handled by a complimentary [Let's Encrypt](https://letsencrypt.org/) service integration, which automatically generates and renews a certificate based on the domain you're using.
 
+> **Note**
+>
+>Container-as-a-Service platforms (for example, Azure Container Apps, DigitalOcean App Platform) include TLS endpoint management, so you do not need to use our optional certificate manager component.
```

Comment:
To match the changes above, but for this context I've removed the examples since they were just mentioned a few paragraphs before:

```suggestion
>Containers as a service, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to use the optional certificate manager component.
```

Reply:
Same as similar comment above (prefer we keep the word platforms here).

Reply:
Combining above suggestions

```suggestion
>Containers as a service platforms, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to use the optional certificate manager component.
```